bundles/org.openhab.binding.lghombot/src/main/java/org/openhab/binding/lghombot/internal/LGHomBotHandler.java

```diff
@@ -0,0 +1,722 @@
+                        break;
+                    case "LGSRV_MEMUSAGE":
+                        if (!value.equals(currentSrvMem)) {
+                            currentSrvMem = value;
+                            channel = new ChannelUID(getThing().getUID(), CHANNEL_SRV_MEM);
+                            updateState(channel, StringType.valueOf(value));
+                        }
+                        break;
+                    case "JSON_TURBO":
+                        OnOffType turbo = value.equalsIgnoreCase("true") ? OnOffType.ON : OnOffType.OFF;
+                        if (!turbo.equals(currentTurbo)) {
+                            currentTurbo = turbo;
+                            channel = new ChannelUID(getThing().getUID(), CHANNEL_TURBO);
+                            updateState(channel, turbo);
+                        }
+                        break;
+                    case "JSON_REPEAT":
+                        OnOffType repeat = value.equalsIgnoreCase("true") ? OnOffType.ON : OnOffType.OFF;
+                        if (!repeat.equals(currentRepeat)) {
+                            currentRepeat = repeat;
+                            channel = new ChannelUID(getThing().getUID(), CHANNEL_REPEAT);
+                            updateState(channel, repeat);
+                        }
+                        break;
+                    case "JSON_MODE":
+                        if (!value.equals(currentMode)) {
+                            currentMode = value;
+                            channel = new ChannelUID(getThing().getUID(), CHANNEL_MODE);
+                            updateState(channel, StringType.valueOf(value));
+                        }
+                        break;
+                    case "JSON_NICKNAME":
+                        if (!value.equals(currentNickname)) {
+                            currentNickname = value;
+                            channel = new ChannelUID(getThing().getUID(), CHANNEL_NICKNAME);
+                            updateState(channel, StringType.valueOf(value));
+                        }
+                        break;
+                    case "CLREC_LAST_CLEAN":
+                        try {
+                            final String stringDate = value.substring(0, 19);
+                            LocalDateTime localDateTime = LocalDateTime.parse(stringDate, formatterLG);
+                            ZonedDateTime date = ZonedDateTime.of(localDateTime, ZoneId.systemDefault());
+                            DateTimeType lastClean = new DateTimeType(date);
+                            if (!lastClean.equals(currentLastClean)) {
+                                currentLastClean = lastClean;
+                                channel = new ChannelUID(getThing().getUID(), CHANNEL_LAST_CLEAN);
+                                updateState(channel, lastClean);
+                            }
+                        } catch (IndexOutOfBoundsException | DateTimeException e) {
```

Comment:
`IndexOutOfBoundsException` is an exception which you should not allow to happen, please just check the String length before doing the substring.